docs/reference/introduction.md

```diff
@@ -1,5 +1,7 @@
 ```{eval-rst}
-.. _reference-index:
```

Comment:
When adding the meta information, let's leave any existing stuff already in the eval-rst if one is present. This info is used for table of contents and cross-page linking.